packages/data_tracker/spec/internal/app/models/internal/sale.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module Internal
+  class Sale < ::Internal::ApplicationRecord
+    track_data
```

Comment:
This is so generic it doesn't really mean anything. Can we use a name that has more meaning?

Reply:
@benlangfeld indeed, I didn't like this as well. Suggestions?

Reply:
How do you feel about @wadewinningham's suggestion: `AuditTracker` for the gem, and this helper method could be renamed to `track_audit_data`?

Reply:
/cc @benlangfeld